internal/x/cobra/cmd.go

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2017-present SIGHUP s.r.l All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package cobrax
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+// GetFullname returns the hierarchy of the command and its parents. For example: "<command> <subcommand>...".
+func GetFullname(c *cobra.Command) string {
```

Comment:
nice one. it could also be nice to expand on this in the future with something like a `superhelp` command that prints all the commands and subcommands in one go.